ui/component/or-map/src/mapwidget.ts

```diff
@@ -873,6 +911,15 @@ export class MapWidget {
         }
     }
 
+    public async getGeoJsonFeature(query: {query: string}){
+        // let gc: MaplibreGeocoder. = this._geocoder;
+    }
+    public async getAddress(query: {lat: number, lon: number}){
+        // let gc: MaplibreGeocoder. = this._geocoder;
+        return this._reverseGeocode(query);
+    }
+
+
```

Comment:
Maybe should be exposed from the `or-map` so no dependency on the map for geocoding